build.cake

```diff
@@ -903,11 +903,59 @@ Task("Android100")
                     .WithProperty("AndroidTargetFrameworks", "MonoAndroid10.0"));
     });
 
+
```

Comment:
This provides a set of targets you can run that clean the obj/bin folders, compile the slns, and then open them in VS ready to rock